.gitignore

```diff
@@ -5,3 +5,4 @@ dev-demo
 .DS_STORE
 .idea
 test.html
+dist
```

Comment:
my reasoning for checking in the `dist` directory is that I want to let people quickly grab the minified library even if they don't want to use NPM. I'm aware there are now CDN services such as `unpkg.com` but they're still effectively NPM proxies. So I'd suggest keeping the `dist` folder checked in, if you don't mind.